xray_vision/backend/mpl/cross_section_2d.py

```diff
@@ -676,3 +774,12 @@ def autoscale_horizontal(self, enable):
     @auto_redraw
     def autoscale_vertical(self, enable):
         self._ax_v.autoscale(enable=False)
+
+
+def _swap_sides(key, enum):
```

Comment:
can you add a comment explaining what this is doing and why?

Reply:
- I will add a comment!

Reply:
This is a helper function to easily swap the sides, e.g. `'left'` -> `'right'` and vice versa. In case the supplied value `key` is not in keys of `enum`, the default value from the `enum` is used (default is the one which has the `True` value).

@kalebswartz7, please create a docstring explaining this and the arguments to the function. Thanks!